shoop/admin/templates/shoop/admin/contact_groups/edit.jinja

```diff
@@ -12,15 +12,19 @@
             {% endcall %}
             {% if contact_group.pk %}
                 {% call content_block(_("Members"), "fa-users") %}
-                    {# TODO: This is very bare-bones; should probably have UI to add/remove members etc... #}
-                    <ul>
-                    {% for member in contact_group.members.all() %}
-                        {% set u=shoop_admin.model_url(member, default="") %}
-                        <li>{% if u %}<a href="{{ u }}">{% endif %}{{ member }}{% if u %}</a>{% endif %}</li>
+                    {% if contact_group.members.exists() %}
```

Comment:
I think the "Edit members"-section should just replace this list. Is there a reason you did a whole new view for editing members?